examples/dart/gen-dart/v1_music/lib/src/f_store_service.dart

```
@@ -584,18 +550,20 @@ class enterAlbumGiveaway_args implements thrift.TBase {
     switch (fieldID) {
       case EMAIL:
         if (value == null) {
-          unsetEmail();
-        } else {
-          this.email = value as String;
+          email = null;
+        } else if (value is String) {
+          email = value;
```

This looks wrong: previously if `value` was not a string then it failed with an error.  Now it silently ignores the invalid value.

---

Addressed.